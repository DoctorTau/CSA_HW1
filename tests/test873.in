195
3059 1211 2891 -5341 -5631 -1727 -1402 9867 6386 2525 -7400 215 -5439 2063 8712 -576 -1485 8553 -8919 -9509 -8435 1789 -4651 7163 5798 -396 -3267 -6605 -2393 553 -7844 -670 1002 -5271 2171 -8261 6293 899 -1095 390 8624 -9127 -4838 -4861 -8024 -699 -1444 8466 2514 5089 -1840 7364 1724 6417 8920 6384 -6826 -1045 8089 -1601 -8820 166 -8125 2489 -8735 7491 -5954 -9725 6647 -3153 -2648 -6305 8123 163 -1595 -4565 -3971 7090 -2258 -2698 5485 3690 -6915 9250 -4817 -9598 8505 1571 -2238 2947 -6508 7992 3103 -6809 -2101 9200 9146 5625 2907 -5548 2271 -1823 -8714 -1086 -9140 5132 -8395 9775 -4128 -6734 -205 -578 2817 -2050 7551 6357 -2689 6232 -1747 9864 -9759 5702 -1931 3208 6776 8734 9085 -6762 6838 -6306 -6997 4995 8485 9945 1148 -3593 9437 -4740 -4257 5321 -5588 3571 -4658 -5661 9803 8642 3639 -3319 -7711 3585 5980 -636 -7439 1547 252 6410 7898 -6510 2145 -3563 -4389 6619 132 6469 -632 -3065 -2397 8330 -6381 981 -7128 2472 2862 -2410 -1540 7351 6984 6346 -6076 -7106 3778 -8204 -8749 -5956 5488 -3694 -3919 6940 9578 863 3086 -8366 8806 -3917 9293 