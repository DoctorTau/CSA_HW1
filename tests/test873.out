-1402 -7400 -576 -396 -7844 -670 -4838 -8024 -1444 -1840 -6826 -8820 -5954 -2648 -2258 -2698 -9598 -2238 -6508 -5548 -8714 -1086 -9140 -4128 -6734 -578 -2050 -6762 -6306 -4740 -5588 -4658 -636 -6510 -632 -7128 -2410 -1540 -6076 -7106 -8204 -5956 -3694 -8366 